code/modules/ruins/lavalandruin_code/syndicate_base.dm

```diff
@@ -63,19 +64,22 @@
 
 /obj/effect/mob_spawn/human/lavaland_syndicate/comms/space
 	flavour_text = "<span class='big bold'>You are a syndicate agent,</span><b> assigned to a small listening post station situated near your hated enemy's top secret research facility: Space Station 13. <b>Monitor enemy activity as best you can, and try to keep a low profile. Do not abandon the base.</b> Use the communication equipment to provide support to any field agents, and sow disinformation to throw Nanotrasen off your trail. Do not let the base fall into enemy hands!</b> \
-	<br><i>You are free to attack anyone not aligned with the Syndicate in the vicinity of your base. <font size=6>DO NOT</font> work against Syndicate personnel (such as traitors or nuclear operatives). You may work with or against non-Syndicate antagonists on a case-by-case basis. <font size=6>DO NOT</font> leave your base without admin permission.</i>" 
+	<br><i>You are free to attack anyone not aligned with the Syndicate in the vicinity of your base. <font size=6>DO NOT</font> work against Syndicate personnel (such as traitors or nuclear operatives). You may work with or against non-Syndicate antagonists on a case-by-case basis. <font size=6>DO NOT</font> leave your base without admin permission.</i>"
 
 /obj/effect/mob_spawn/human/lavaland_syndicate/comms/space/Initialize(mapload)
 	. = ..()
-	if(prob(90)) //only has a 10% chance of existing, otherwise it'll just be a NPC syndie.
-		new /mob/living/simple_animal/hostile/syndicate/ranged(get_turf(src))
+	if(!GAMEMODE_IS_TRAITOR || prob(90)) // must be a mode with traitors, and roll 10% chance, to allow this
```

Comment:
This is a 90% chance to roll if the gamemode is not traitors, you want prob(10) for a ten percent chance the previous line rolled 90% chance to spawn just an NPC

Reply:
I *think* you're misreading it. It should "return INITIALIZE_HINT_QDEL" 100% of the time in non-traitor game modes, and 90% of the time in traitor modes. So it only persists (and thus, is active) 10% of the time in traitor modes.
The logic is equivalent to:
if (!GAMEMODE_IS_TRAITOR)
 return INITIALIZE_HINT_QDEL
if (prob(90))
 return INITIALIZE_HINT_QDEL
The INITIALIZE_HINT_QDEL means the object is deleted and not available for use by players.
If I'm wrong, let me know, but that's how it should (and I think, does) work.

Reply:
Right, I misread the probability part but i think you still want to use &&(and) instead of ||(or) because if it’s traitors it will be there 100%, but only have a 10% chance to spawn on non-traitor rounds(only one of the values have to be true for it to run the code underneath the or both values would need to be true if you used and which in this situation I think you want)

Reply:
Right. Syndie comms agents shouldn't spawn at all if there are no traitors (since they have nobody to work with) and the code now enforces this.